server/server.go

```diff
@@ -49,13 +49,13 @@ type OIDCServer interface {
 	Login(oidc.Identity, string) (string, error)
 
 	// CodeToken exchanges a code for an ID token and a refresh token string on success.
-	CodeToken(creds oidc.ClientCredentials, sessionKey string) (*jose.JWT, string, error)
+	CodeToken(creds oidc.ClientCredentials, sessionKey string) (*jose.JWT, string, int64, error)
```

Comment:
Instead of int64, please return a `time` package type to indicate the exact meaning of the returned value. For instance, you can return a `time.Time` here then convert using `int64(time.Now().Sub(t))` to "expired at".